packages/salesforcedx-vscode-apex/README.md

```diff
@@ -10,13 +10,15 @@ Before you set up this extension, make sure that you have these essentials.
   Some features in Salesforce Extensions for VS Code depend upon the Java 8 Platform, Standard Edition Development Kit (JDK).  
   If you don’t already have the JDK installed, install the latest version of the Java 8 JDK from [Java SE Development Kit 8 Downloads](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).  
 * **[Visual Studio Code](https://code.visualstudio.com/download) v1.17 or later**  
+* **[Salesforce Extensions for VS Code](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)**  
+  Code smartness for sObjects in Apex code is powered by the [salesforcedx-vscode-core](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core) extension. We suggest that you install all extensions in the [salesforcedx-vscode](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) extension pack.
 
 ## View Code-Completion Suggestions
 To see code-completion suggestions, press Ctrl+space when you’re working in a `.cls` or `.trigger` file. To navigate between the suggestions, use the arrow keys. To auto-complete a suggestion from the list, press Enter.  
 ![Animation showing code completion of a System.debug() statement](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/images/apex_completion.gif)
 
 ## View or Jump to Definitions
-You can preview, view, or go to definitions of Apex methods, properties, constructors, and local and class variables.  
+You can preview, view, or go to definitions of Apex methods, properties, constructors, local and class variables, and the standard and custom objects in your default scratch org. (See the "Enable Code Smartness for SObjects" section of this README for information on working with standard and custom objects.)  
```

Comment:
Custom fields of these objects as well, but maybe this is getting to be a bit long.